README.md

```diff
@@ -182,25 +182,25 @@ Or, using Sass:
 
 #### Theme modifiers
 
-There are 5 named themes in oButtons. oButtons also allows developers to create custom themes using o-colors names.
-
-[View demos](http://registry.origami.ft.com/components/o-buttons)
+There are named themes in oButtons. oButtons also allows developers to create custom themes using o-colors names.
```

Comment:
100% pedantic, but if we aren't refering to the actual `oButtons` JS class, then we should use `o-buttons`?

Reply:
🤔 it's kind of both here, isn't it?